en/drive_client/drive_client_2.0_for_windows_10.md

```diff
@@ -28,14 +30,34 @@ You can control which files or folder are cached locally. This can be changed fr
 
 ![seadrive-cache-control](imgs/seadrive-cache-control.png)
 
-In SeaDrive 1.0, cached files are not automatically updated when they're updated on the server. In SeaDrive 2.0, full and pinned files are automatically kept in sync with the server.
+In SeaDrive 1.0, cached files are not automatically updated when they're updated on the server. In SeaDrive 3.0, full and pinned files are automatically kept in sync with the server.
 
 ## File Download and Control
 
 Whenever you open a placeholder file, the operating system will automatically start to download it. If the file may take some time to download, there will be a progress bar shown up in file explorer and you may cancel the download.
 
 Sometimes a background application may try to download a file in the virtual drive (such as an Anti-Virus software). You will be notified by the operating system about this and you may choose to cancel the download or disallow the application from automatically downloading files in the future.
 
+## Cache Management
+
+You can customize the cache directory and set the cache size limit and clean interval in the settings.
+
+When the cache size surpasses the limit, SeaDrive will automatically free up space by prioritizing files based on their last modification time. This process continues until the cache size is reduced to below 70% of the specified limit.
+
+![seadrive-cache-management](imgs/seadrive-cache-management.png)
+
+## Account Management
+
+In SeaDrive 3.0, you can log into multiple accounts across different servers, providing access to files from all linked accounts within the cache directory.
+
+![seadrive-account-management](imgs/seadrive-account-management.png)
+
+You can log out of the account by selecting 'Delete' in the account management, at this time it will log out of the account and keep the local files that have been cached. 
+
+When the library contains invalid placeholders, you can use the function of resync account to fix this issue. SeaDrive retains the old cache directory while generating new placeholders in a fresh cache directory.
```

Comment:
Some special sync errors, such as invalid placeholders, local metadata corruption, can be fixed by re-syncing the account.